src/App.css

```diff
@@ -1,51 +1,142 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+  font-family: 'Montserrat', sans-serif;
+}
+
+.header {
+  text-transform: uppercase;
+  margin: 8rem 0 0 2rem;
+  font-size: 2rem;
+  color: #888;
+  font-weight: bold;
+}
+
+button {
+  padding: 1rem;
+  background-color: #0290ff;
+  width: 5rem;
+  height: 3rem;
+  color: #fff;
+  font-family: Montserrat, sans-serif;
+  font-size: large;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#root {
+  padding: 3rem 5rem;
+  height: max-content;
+}
+
+.app {
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #fafafa;
+  border-radius: 4px;
+  padding: 0 0 4rem;
 }
 
 li,
 a {
   text-decoration: none;
-  font-size: 2rem;
+  text-transform: uppercase;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
 }
 
 nav {
   display: flex;
-  justify-content: space-around;
+  justify-content: flex-start;
+  gap: 3rem;
   align-items: center;
-  padding: 0 5rem;
-  gap: 35rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  height: 6rem;
+  background-color: #fff;
 }
```

Comment:
- [ ] I recommend removing the margins from the top, left, and right of your nav so your design matches the provided template design.

<img width="953" alt="image" src="https://user-images.githubusercontent.com/77190203/162591171-deca7d8f-0a29-4c8e-baff-bcea69915614.png">